drag_and_drop_v2/default_data.py

```diff
@@ -72,7 +74,7 @@
             "displayName": ITEM_TOP_ZONE_NAME,
             "feedback": {
                 "incorrect": ITEM_INCORRECT_FEEDBACK,
-                "correct": ITEM_CORRECT_FEEDBACK.format(zone=TOP_ZONE_TITLE)
```

Comment:
@pomegranited , I changed these templated strings to static strings to fix the issue of default data. Hopefully it will not cause problems with translations and don't look that bad either.

Reply:
@pomegranited, I have kept this as a separate commit and not extracted the strings into po/mo since I did not know how to do that. perhaps if you could guide me. If it's not required, then I will remove this commit.

Reply:
Thanks @afzaledx -- that takes care of the last bit of untranslated text shown on the default demo DnDv2, so this is ok to keep.

Note though that your new strings aren't extracted/compiled into the `.po`/`.mo`/`text.js` files yet, so won't appear translated.  Unfortunately the process for doing this is still very manual, and so this is out of scope here.  We're discussing whether we can update [edx:i18n-tools](https://github.com/edx/i18n-tools) to support managing XBlock translations. CC @bradenmacdonald